spec.bs

```diff
@@ -1712,8 +1677,13 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
           1. Let |directFromSellerSignalsForBuyer| be the result of running
             [=get direct from seller signals for a buyer=] with |directFromSellerSignals|, and |ig|'s
             [=interest group/owner=].
+          1. Let |dataVersion| be [=map/get=] |trustedBiddingSignalsBatcher|'s [=trusted bidding signals batcher/data versions=]

```

Comment:
Yeah this is wrong. What variable is the map, and which variable is the key? You need to specify those. Then, just do `Let value be map[key]`.

Reply:
|trustedBiddingSignalsBatcher|'s [=trusted bidding signals batcher/data versions=] is the map. 
|ig|'s [=interest group/name=] is the key.  So I just... drop the [=map/get=] ?

Reply:
I would just write `|trustedBiddingSignalsBatcher|'s [=trusted bidding signals batcher/data versions=][|ig|'s [=interest group/name=]]` yeah, although I might pull some of those out into separate variables for readability.. But yeah literal map bracket syntax works just fine.

Reply:
Done